t/00-report.t

```diff
@@ -67,7 +67,7 @@ my $len = 0;
 for my $dep (@depends) {
     my $l = length($dep);
     $len = $l if $l > $len;
-    $deps{$dep} = eval "require $dep; $dep->VERSION" || "N/A";
+    $deps{$dep} = eval "require $dep" ? ($dep->VERSION || '0') : 'N/A';
```

Comment:
Why does this need to be changed for 5.6?

Reply:
Ignore this comment. I get it.

Reply:
No, it is not required for 5.6. But some modules (e.g. Carp) do not set `our $VERSION` and therefore that `$dep->VERSION` call returns empty string which is in boolean context treated as false and so `"N/A"` was used. This change is just for diagnostic purpose for any perl version to distinguish between fact that module was not loaded and fact that version of module was not specified.